projects/_posts/2015-01-30-Snake-Game.md

```diff
@@ -0,0 +1,21 @@
+---
+# Fill out as many of these as you can, and delete the rest.
+author: [ "@donwojtallo", "Wojciech Ałaszewski" ] # The person(s) who created this project. 
+# Any author that starts with "@" will pull information from github.
+programming_language: "JavaScript" # The programming language used in this project
+
+youtube_video_id: kHUkGucSeU0 # The video ID of the YouTube video to be displayed with this post
+
+project_homepage_url: "https://github.com/donwojtallo/ev3dev-snake-js" # Homepage for this project
+source_code_url: "https://github.com/donwojtallo/ev3dev-snake-js" # Provide a link to your code
+building_instructions_url: "https://github.com/donwojtallo/ev3dev-snake-js"
```

Comment:
This is actually supposed to be for LEGO building instructions, not software building instructions.